src/content-helpers.ts

```diff
@@ -189,12 +189,14 @@ export type MakeTopicContent = (topic: string | null | undefined, htmlTopic?: st
 
 export const makeTopicContent: MakeTopicContent = (topic, htmlTopic) => {
     const renderings = [];
-    if (isProvided(topic)) {
-        renderings.push({ body: topic, mimetype: "text/plain" });
-    }
+    // Put HTML first because clients will render the first type in
+    // the array that they understand
```

Comment:
This also needs to be reflected in `TopicState` so that users of the SDK actually know which rendering came first. Maybe we could just add a `preferred` flag into the type?

```
export type TopicState = {
    text?: string;
    html?: string;
    preferred?: "text" | "html"
};
```

This won't scale well but I don't realistically foresee further mimetypes being added here in the near future.